Fabric.Authorization.API/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/aspnetcore:1.1.2
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2
```

Comment:
dont know if it matters, but looks like 2.1 is LTS but 2.2 is current.  Do we want to use the 2.1 release instead?  here is what I saw this on:  https://hub.docker.com/_/microsoft-dotnet-core-aspnet

Reply:
It's a good point, we could back down to .net core 2.1.x, this will give us a longer runway to upgrade to 3.x.